Jenkinsfile-Release

```diff
@@ -39,6 +49,14 @@ dockerizedBuildPipeline(
       sh 'git push'
     }
   },
+  postDeploy: {
+    def chart = params.chart
+    def indexFile = readYaml file: 'docs/index.yaml'
+    def version = indexFile.entries[chart][0].version
+
+    jira.versionPrefix = jiraVersionMappings[chart]
+    jiraSetFixVersion(jira, version)
```

Comment:
IIUC the code under `postDeploy` is executed on all builds, even failed ones. If that's the case (please check), the new code you added under `postDeploy` should go under `onSuccess` as the last thing we do.

Reply:
oh, ok, I am going to check and update accordingly

Reply:
@eduard-tita qq, I see the `dockerizedBuildPipeline` uses `postDeploy` on a stage after the `deploy` stage,  with that in mind I understand that if any of the stages before this one fails, then that stage will not be executed, is that understanding ok? If yes, then I think we should be ok with the current code.
Check this link: https://github.com/sonatype/jenkins-shared/blob/b9726e1f2ecbb15339f269839e62dc99b744d208/vars/dockerizedBuildPipeline.groovy#L100

Reply:
Thanks for checking, @hectorhuol. My assumption was wrong. The code is fine where it is now.